<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshToLayerDialog</class>
 <widget class="QDialog" name="MeshToLayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>146</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>347</width>
    <height>146</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>347</width>
    <height>146</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lMesh">
       <property name="text">
        <string>Mesh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbMesh"/>
     </item>
     <item>
      <widget class="QLabel" name="lLayer">
       <property name="text">
        <string>Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbLayer"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lProcess">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pbTransfer">
     <property name="text">
      <string>Transfer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
